app/assets/v2/js/shared.js

```diff
@@ -749,24 +749,9 @@ var currentNetwork = function(network) {
       $('#navbar-network-banner').addClass('hidden');
     } else {
       if (!network) {
-        info = gettext('Web3 disabled. Please install ') +
-          '<a href="https://metamask.io/?utm_source=gitcoin.co&utm_medium=referral" target="_blank" rel="noopener noreferrer">Metamask</a>';
-        $('#current-network').text(gettext('Metamask Not Enabled'));
-        $('#navbar-network-banner').html(info);
-      } else if (network == 'locked') {
-        if (is_metamask_approved || !is_metamask_unlocked) {
-          info = gettext('Web3 locked. Please unlock ') +
-            '<a href="https://metamask.io/?utm_source=gitcoin.co&utm_medium=referral" target="_blank" rel="noopener noreferrer">Metamask</a>';
-          $('#current-network').text(gettext('Metamask Locked'));
-          $('#navbar-network-banner').html(info);
-        } else if (window.ethereum._metamask) {
-          info = gettext('Metamask not connected. ') +
-              '<button id="metamask_connect" onclick="approve_metamask()">Click here to connect to metamask</button>';
-          $('#current-network').text(gettext('Metamask Not Connected'));
-          $('#navbar-network-banner').html(info);
-        }
+        $('#current-network').text(gettext('Fortmatic Not Enabled'));
```

Comment:
What is the plan with this?
So lets plan on this to we can get to a right place as we are doing another stuff also with crosschain this code will not getting in.
The solution is something like https://www.myetherwallet.com/access-my-wallet or https://web3connect.com/ where people can decide what to use to connect or use noting (for crosschain)

Reply:
The simplest idea that I had in mind was simply to always use Metamask if it exists, otherwise redirect them to use Fortmatic. I do agree that it makes more sense to prompt the user to select the wallet, but from a UX perspective, this might get annoying very fast if I had to choose a wallet every time the browser can't see a `window.ethereum` object defined (which is quite often, since the document.location.reload called from `listen_for_web3_changes` happens frequently).